styles/components/_app-grid.css

```diff
@@ -1,27 +1,28 @@
 .app-grid {
   display: grid;
-  grid-template-rows: 48px auto 48px;
-  height: 100vh;
-}
-
-.app-grid__header {
-  font-size: medium;
-  background-color: antiquewhite;
-}
-
-.app-grid__main {
+  grid-template-columns: 1fr;
+  grid-template-rows: 1fr 1fr 1fr;
   overflow-y: scroll;
   background-color: slategray;
 }
 
-.app-grid__main__card {
-  border: solid 1px black;
+.app-grid__card {
+  position: relative;
+  /* height: 300px;
```

Comment:
We always want the main branch to be as clean as possible. This means, that code, which is not used, should be deleted.